packages/themes/pennwell/styles/components/_item.scss

```diff
@@ -103,13 +103,16 @@
   &__footer {
     @include theme-small-text();
     display: flex;
-    flex-direction: row;
+    flex-direction: column;
     justify-content: space-between;
     margin-top: auto;
     color: $gray;
     &:empty {
       display: none;
     }
+    @include media-breakpoint-up("md", $grid-breakpoints) {
```

Comment:
You can simply call `@include media-breakpoint-up(md)` as the `$grid-breakpoints` var is already the default value of the mixin's second argument. I realize some other areas in the code don't follow this format, but I'm in the process of converting them. :)

Reply:
done